src/Generator/JekyllPageGenerator.php

```diff
@@ -0,0 +1,36 @@
+<?php
+declare(strict_types=1);
+
+namespace App\Generator;
+
+use App\Value\Sniff;
+
+class JekyllPageGenerator extends MarkdownGenerator implements Generator
```

Comment:
Classes should be final and we should not extend implementations. It's okay to copy an existing to make the necessary changes. We can always extract truly reusable elements and inject them as dependencies instead. The test for the new class should encompass the common elements as well, otherwise we're not truly testing them, and any unintended regressions would go unnoticed.

Reply:
Let's defer it and create a ticket to improve later.